sdk/src/tx/txParamProcessor.ts

```diff
@@ -49,10 +50,15 @@ export class TransactionParamProcessor {
 			const computeUnits = await this.getComputeUnitsFromSim(simTxResult);
 
 			// Apply the buffer, but round down to the MAX_COMPUTE_UNITS, and round up to the nearest whole number
-			const bufferedComputeUnits = Math.ceil(
+			let bufferedComputeUnits = Math.ceil(
 				Math.min(computeUnits * bufferMultiplier, MAX_COMPUTE_UNITS)
 			);
 
+			// If a lower bound CU is passed then enforce it
+			if (lowerBoundCu) {
+				bufferedComputeUnits = Math.max(bufferedComputeUnits, lowerBoundCu);
```

Comment:
something small that could be inconsequential; but `lowerBoundCu` could potentially be higher than `MAX_COMPUTE_UNITS`, which then sets `bufferedComputeUnits` to an invalid CU. a simple fix could be to just check if `lowerBoundCu` is less than `MAX_COMPUTE_UNITS`